packages/storybook/package.json

```diff
@@ -26,20 +26,13 @@
     "@ni/spright-components": "*",
     "@ni/spright-react": "*",
     "@rollup/plugin-node-resolve": "^16.0.0",
-    "@storybook/addon-a11y": "^8.4.3",
-    "@storybook/addon-actions": "^8.4.3",
-    "@storybook/addon-docs": "^8.4.3",
-    "@storybook/addon-essentials": "^8.4.3",
-    "@storybook/addon-interactions": "^8.4.3",
-    "@storybook/addon-links": "^8.4.3",
+    "@storybook/addon-a11y": "9.0.5",
+    "@storybook/addon-docs": "9.0.5",
+    "@storybook/addon-links": "9.0.5",
     "@storybook/addon-webpack5-compiler-swc": "^3.0.0",
```

Comment:
There are webpack related things here and in main.js, should be cleaning up things related to webpack